modules/custom/az_migration/migrations/az_node_carousel.yml

```
@@ -95,18 +95,23 @@ process:
   changed: changed
   promote: promote
   sticky: sticky
-
   path/pathauto:
     -
-      plugin: skip_on_empty
+      plugin: str_replace
+      regex: true
       source: alias
-      method: process
+      search: /.*$/
+      replace: 0
     -
       plugin: default_value
-      default_value: 0
-
-  path/alias:
+      default_value: 1
```

```suggestion
      strict: true
      default_value: 1
```
https://api.drupal.org/api/drupal/core%21modules%21migrate%21src%21Plugin%21migrate%21process%21DefaultValue.php/class/DefaultValue/9